client/src/components/SearchForm.js

```diff
@@ -52,6 +55,14 @@ class SearchForm extends React.Component {
   triggerSearch() {
     const p = this.props;
     p.onSearch();
+
+    const query = this.props.search.query;
+    const queryFilters = this.buildQuery(this.props.search.filters, true)
+
+    this.history.push({
```

Comment:
I'd actually prefer this be `replace` instead of `push`, but it's personal preference.

Reply:
You use push in Navbar.js?

Reply:
Just tested to make sure but replace doesn't push to the history stack so you can't press the back button and go to your previous query. So I think that push would be the best option.

Reply:
That's actually precisely why I want replace in this instance--if you're using the search function normally, then I don't think your nav history should get cluttered up with previous searches.

Reply:
I would think that you want to press back and go to previous searches, pretty much like any search engine: Google, etc.
But if you think replace is the better option I'll change it :)